src/component/nuclear/card/TheatreCard.js

```diff
@@ -0,0 +1,128 @@
+import React, { useState } from 'react';
+import SeatSelectionComponent from '../../seat/SeatSelectionComponent'; // Ensure you import the SeatSelectionComponent
+
+const TheatreCard = ({ movieData }) => {
+  const theatreDetails = movieData.theatredetail;
+
+  // Function to get the current date in YYYY-MM-DD format
+  const getCurrentDate = () => {
+    const today = new Date();
+    const year = today.getFullYear();
+    const month = String(today.getMonth() + 1).padStart(2, '0'); // Months are 0-based, so +1
+    const day = String(today.getDate()).padStart(2, '0');
+    return `${year}-${month}-${day}`;
+  };
```

Comment:
Can we keep this in any common class file, So we can use this method from anywhere